company/okrs/2020_q1.md

```diff
@@ -0,0 +1,91 @@
+# 2020-Q1 OKRs
+
+## CEO
+
+1. **CEO: Grow ARR consistently**
+   1. CEO KR: [$N][N] net new ARR
+   1. CEO KR: Sales leadership owns revenue targets
+   1. CEO KR: Sales, engineering, and product have a successful process for prioritization
+1. **CEO: Aspirational, efficient all-remote team**
+   1. CEO KR: Roles and responsibilities are documented and maintained in handbook
+   1. CEO KR: Handbook-first
+
+## Sales and marketing
+
+1. **Sales (Closing Deals): Achieve [$N][N] net new ARR.**
+   1. Sales KR: Have each AE close 3 Tier 1 deals (>=$100K ARR) and 10 non-Tier 1 deals, 2 of which are from their own outbound outreach efforts.
+   1. Sales KR: Get 70% of companies on a free trial to convert to paid customers.
+1. **Sales (Pipeline Optimization): Scale our sales pipeline to 50+ deals per AE, of which 60% of deals are engaged and actively evaluating Sourcegraph.**
+   1. Sales KR: Round robin new instances across team to collectively achieve a 15%+ response rate from Prospects who set up a new Sourcegraph instance.
+
+1. **VP Business: Build a Customer Success organization that supports and helps us exceed our ARR goals.**
+   1. VP Business (Success) KR: Generate at least [$M][M] in net new ARR from existing accounts.
+   1. VP Business (Success) KR: Hire a Head of Customer Success who can own this function.
+   1. VP Business (Success) KR: Reach monthly NPS of 60 at all Tier 1 customers.
+
+1. **VP Business: Make Sourcegraph a lean organization that uses budgets to guide decision making.**
+
+1. **VP Marketing: Deliver sufficient MQLs to sales to reach [$N][N] net new ARR goal.**
+1. **VP Marketing: Create new category of developer tool.**
+1. **VP Marketing: Become the dominant provider of source code search.**
+
+## Product
+
+1. **Product: Deliver on product roadmap**
+   1. Product KR: Roadmap communicates rough timeline for upcoming work.
+   1. Product KR: Each feature is reviewed before release to ensure quality UX.
+1. **Product: Reduce product friction to accelerate customers through pipeline**
```

Comment:
One recurring thought I had in the past week: what about documentation?

We _have_ a lot of documentation, but I think it's not the most navigatable docs page (navbar on the left doesn't match document structure, but lists subpages, which are sometimes listed, sometimees not, etc.) and it has rendering glitches and it's often a mix between "guide" and "reference".

If we look to some documentation pages that I would personally consider "best in class" (https://stripe.com/docs, https://www.terraform.io/docs/index.html) we can get a ton of inspiration for how to improve our documentation.

I do think there's a possibility that docs add needless "friction" and might cause customers to drop out of the pipeline.

Reply:
Huge 👍 to this.

Reply:
We should find an owner for improving the docs (See [RFC 102](https://docs.google.com/document/d/1rmNEiyU-gP97BYVwZWVWo7G94jeRAtrQrk4VZGlcBic/edit))